RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -7,12 +7,170 @@
 
 import SwiftUI
 
+/// Player control
 struct ContentView: View {
+    // MARK: - Private Constants
+    private enum Constants {
+        static let emptyString = ""
+        static let posterWidthHeight: CGFloat = 250
+        static let posterCornerRadius: CGFloat = 50
+        static let shadowRadius: CGFloat = 16
+        static let shadowX: CGFloat = 7
+        static let shadowY: CGFloat = 7
+        static let rewindButtonImageName = "backward"
+        static let buttonLoWidth: CGFloat = 50
+        static let buttonHiWidth: CGFloat = 75
+        static let buttonHeight: CGFloat = 35
+        static let buttonCornerRadius: CGFloat = 15
+        static let playButtonName = "Play"
+        static let pauseButtonName = "Pause"
+        static let stopButtonName = "Stop"
+        static let forwardButtonImageName = "forward"
+        static let saveTrackName = "Save track"
+        static let savedTrackName = "Track saved"
+        static let okButtonName = "Ok"
+        static let shareTextName = "Share..."
+        static let shareTrackName = "Share track"
+        static let cancelName = "Cancel"
+        static let confirmMessage = "Are you sure?"
+    }
+    // MARK: - Private ObservedObject Properties
```

Comment:
```suggestion
    // MARK: - Private Properties
```
 // MARK: - Public Properties
  такие марки надо